py/README.md

```diff
@@ -0,0 +1,3 @@
+# Serial Keel Client for Python 3
+
+## TODO
```

Comment:
I was thinking that maybe, we could elaborate here in the README based on the sample mock test that we have.